docker/datahub-gms/start.sh

```diff
@@ -57,7 +57,6 @@ fi
 # kubernetes deployments
 auth_resource_dir=${AUTH_RESOURCES_DIR:-"/etc/datahub/plugins/auth/resources"}
 # Option --classes ${AUTH_RESOURCE_LOOK_UP_DIR} is added for Apache Ranger library to load the ranger-datahub-security.xml from classpath
-mkdir -p $auth_resource_dir
```

Comment:
Do we need this in both places? 

If the user is mounting from a local directory, will this still work?

Reply:
I think it would work. Adding here does not work because this script runs as datahub user which does not have the correct rights.